kore/src/main/scala/org/kframework/attributes/Att.scala

```diff
@@ -179,113 +184,123 @@ object Att {
+  final val HYBRID = Key.builtin("hybrid", KeyParameter.Optional, onlyon[Production])
+  final val IDEM = Key.builtin("idem", KeyParameter.Forbidden, onlyon[Production])
+  final val IMPURE = Key.builtin("impure", KeyParameter.Forbidden, onlyon[Production])
+  final val INDEX = Key.builtin("index", KeyParameter.Required, onlyon[Production])
+  final val INITIAL = Key.builtin("initial", KeyParameter.Forbidden, onlyon[Production])
+  final val INITIALIZER = Key.builtin("initializer", KeyParameter.Forbidden, onlyon2[Production, Rule])
+  final val INJECTIVE = Key.builtin("injective", KeyParameter.Forbidden, onlyon[Production])
+  final val INTERNAL = Key.builtin("internal", KeyParameter.Forbidden, onlyon[Production])
+  final val KAST = Key.builtin("kast", KeyParameter.Forbidden, onlyon[Module])
+  final val KLABEL = Key.builtin("klabel", KeyParameter.Required, onlyon[Production])
+  final val KORE = Key.builtin("kore", KeyParameter.Forbidden, onlyon2[RuleOrClaim, Module])
+  final val LABEL = Key.builtin("label", KeyParameter.Required, onlyon[Sentence])
+  final val LATEX = Key.builtin("latex", KeyParameter.Required, onlyon[Production])
+  final val LEFT = Key.builtin("left", KeyParameter.Optional, onlyon[Production])
+  final val LOCATIONS = Key.builtin("locations", KeyParameter.Forbidden, onlyon[SyntaxSort])
+  final val MACRO = Key.builtin("macro", KeyParameter.Forbidden, onlyon2[Production, Rule])
+  final val MACRO_REC = Key.builtin("macro-rec", KeyParameter.Forbidden, onlyon2[Production, Rule])
+  final val MAINCELL = Key.builtin("maincell", KeyParameter.Forbidden, onlyon[Production])
+  final val MEMO = Key.builtin("memo", KeyParameter.Forbidden, onlyon[Production])
+  final val ML_BINDER = Key.builtin("mlBinder", KeyParameter.Forbidden, onlyon[Production])
+  final val ML_OP = Key.builtin("mlOp", KeyParameter.Forbidden, onlyon[Production])
+  final val MULTIPLICITY = Key.builtin("multiplicity", KeyParameter.Required, onlyon[Production])
+  final val NON_ASSOC = Key.builtin("non-assoc", KeyParameter.Forbidden, onlyon[Production])
+  final val NON_EXECUTABLE = Key.builtin("non-executable", KeyParameter.Forbidden, onlyon[Rule])
+  final val NOT_LR1 = Key.builtin("not-lr1", KeyParameter.Forbidden, onlyon[Module])
+  final val NO_EVALUATORS = Key.builtin("no-evaluators", KeyParameter.Forbidden, onlyon[Production])
+  final val ONE_PATH = Key.builtin("one-path", KeyParameter.Forbidden, onlyon[Claim])
+  final val OWISE = Key.builtin("owise", KeyParameter.Forbidden, onlyon[Rule])
+  final val PARSER = Key.builtin("parser", KeyParameter.Required, onlyon[Production])
+  final val PREC = Key.builtin("prec", KeyParameter.Required, onlyon[Production])
+  final val PREFER = Key.builtin("prefer", KeyParameter.Forbidden, onlyon[Production])
+  final val PRESERVES_DEFINEDNESS = Key.builtin("preserves-definedness", KeyParameter.Forbidden, onlyon[Rule])
+  final val PRIORITY = Key.builtin("priority", KeyParameter.Required, onlyon2[Production, Rule])
```

Comment:
Any particular reason this was marked as allowed on productions?

Reply:
Yeah, there's even a specific commit for this line. There's a k-exercise that has the priority attribute on a configuration declaration cell:

https://github.com/runtimeverification/k/actions/runs/5907674755/job/16025988153#step:7:368

This also made me figure that if `strict` or `seqstrict` are being used, then the user might want to control the priority on the rules that are generated there.

Reply:
Ah I see. Okay. I'm approving this then.